saleor/order/utils.py

```diff
@@ -195,6 +195,8 @@ def update_taxes_for_order_line(
     line: "OrderLine", order: "Order", manager, tax_included
 ):
     variant = line.variant
+    if not variant:
+        return
```

Comment:
Could you add a test for the situation which is fixed?

Reply:
added